src/components/MultiHopTrade/components/MultiHopTradeConfirm/MultiHopTradeConfirm.tsx

```diff
@@ -65,39 +80,56 @@ export const MultiHopTradeConfirm = memo(() => {
   ])
 
   return (
-    <SlideTransition>
-      <Card flex={1} borderRadius={cardBorderRadius} width='full' variant='dashboard'>
-        <CardHeader px={6} pt={4}>
-          <WithBackButton handleBack={handleBack}>
-            <Heading textAlign='center' fontSize='md'>
-              <Text
-                translation={
-                  tradeExecutionState === MultiHopExecutionState.Previewing
-                    ? 'trade.confirmDetails'
-                    : 'trade.trade'
-                }
-              />
-            </Heading>
-          </WithBackButton>
-        </CardHeader>
-        {tradeExecutionState === MultiHopExecutionState.TradeComplete ? (
-          <TradeSuccess handleBack={handleBack}>
-            <Hops isFirstHopOpen isSecondHopOpen />
-          </TradeSuccess>
-        ) : (
-          <>
-            <CardBody py={0} px={0}>
-              <Hops
-                isFirstHopOpen={isFirstHopOpen}
-                isSecondHopOpen={isSecondHopOpen}
-                onToggleFirstHop={onToggleFirstHop}
-                onToggleSecondHop={onToggleSecondHop}
-              />
-            </CardBody>
-            <Footer />
-          </>
-        )}
-      </Card>
-    </SlideTransition>
+    <>
+      <SlideTransition>
+        <Card flex={1} borderRadius={cardBorderRadius} width='full' variant='dashboard'>
+          <CardHeader px={6} pt={4}>
+            <WithBackButton handleBack={handleBack}>
+              <Heading textAlign='center' fontSize='md'>
+                <Text
+                  translation={
+                    tradeExecutionState === MultiHopExecutionState.Previewing
+                      ? 'trade.confirmDetails'
+                      : 'trade.trade'
+                  }
+                />
+              </Heading>
+            </WithBackButton>
+          </CardHeader>
+          {tradeExecutionState === MultiHopExecutionState.TradeComplete ? (
+            <TradeSuccess handleBack={handleBack}>
+              <div ref={targetRef1} />
+            </TradeSuccess>
+          ) : (
+            <>
+              <CardBody py={0} px={0}>
+                <div ref={targetRef2} />
+              </CardBody>
+              <Footer />
+            </>
+          )}
+        </Card>
+      </SlideTransition>
+      <div ref={containerRef}>
+        <Hops
+          isFirstHopOpen={
+            tradeExecutionState === MultiHopExecutionState.TradeComplete || isFirstHopOpen
```

Comment:
We could extract `tradeExecutionState === MultiHopExecutionState.TradeComplete` into `isTradeComplete` for better readability.